.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,105 @@
+name: "🐛 Bug Report"
+description: "Submit a bug report to help us improve"
+title: "🐛 Bug Report: "
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out our bug report form 🙏
+   - type: textarea
+    id: description
+    validations:
+      required: true
+    attributes:
+      label: "📜 Description"
+      description: "A clear and concise description of what the bug is."
+      placeholder: "It bugs out when ..."
+  - type: textarea
+    id: steps-to-reproduce
+    validations:
+      required: true
+    attributes:
+      label: "👟 Reproduction steps"
+      description: "How do you trigger this bug? Please walk us through it step by step."
+      placeholder: "1. When I ..."
+  - type: textarea
+    id: expected-behavior
+    validations:
+      required: true
+    attributes:
+      label: "👍 Expected behavior"
+      description: "What did you think would happen?"
+      placeholder: "It should ..."
+  - type: textarea
+    id: actual-behavior
+    validations:
+      required: true
+    attributes:
+      label: "👎 Actual Behavior"
+      description: "What did actually happen? Add screenshots, if applicable."
+      placeholder: "It actually ..."
+  - type: textarea
+    id: run-logs
+    validations:
+      required: true
+    attributes:
+      label: "❓ Output of the Following"
+      description: "Run the following and paste the output below for a better understanding
+                    1. kubectl get pods -n <openebs_namespace> --show-labels
+                    2. kubectl logs <upgrade_job_pod> -n <openebs_namespace>`"
+      placeholder: "Output/Screenshots here ..."
+  - type: dropdown
+    id: operating-system
+    attributes:
+      label: "💻 Operating system"
+      description: "What OS is your app running on?"
+      options:
+        - Linux
+        - MacOS
+        - Windows
+        - Something else
+    validations:
+      required: true
+  - type: dropdown
+    id: browser
+    attributes:
+      label: "🌍 Browser"
+      description: "What browser is your app running on?"
+      options:
+        - Chrome
+        - Safari
+        - Firefox
+        - Opera
+        - Edge
+        - Something Else
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    validations:
+      required: true
+    attributes:
+      label: "🧱 Environment"
+      description: "Is your environment customized in any way?"
+      placeholder: "- OpenEBS version (use `kubectl get po -n openebs --show-labels`):
+                    - Kubernetes version (use `kubectl version`):
+                    - Cloud provider or hardware configuration:
+                    - OS (e.g: `cat /etc/os-release`):
+                    - kernel (e.g: `uname -a`):
+                    - others:"
```

Comment:
The LocalPV-provisioner is not a regular web-app per say, so things about the **browsers** & **operating systems** don't make sense, but you've added a very good point which can be beneficial for UX(and will likely also play well with bots which create issues **hopefully**) and we might use this for specifying K8s versions like `1.22` or K8s-version ranges.